governator-core/src/main/java/com/netflix/governator/SafeLifecycleListener.java

```diff
@@ -31,12 +31,12 @@ public void onStarted() {
 
     @Override
     public void onStopped(Throwable t) {
-        LOG.info("Failed LifecycleListener '{}'", delegate);
+        LOG.info("Stopping LifecycleListener '{}'", delegate, t);
         try {
             delegate.onStopped(t);
         }
```

Comment:
Might be useful to log a "Stopped LifecycleListenter '{}'" here so we have confirmation that onStopped completed